bodhi/server/config.py

```diff
@@ -460,6 +460,12 @@ class BodhiConfig(dict):
         'pagure_flatpak_namespace': {
             'value': 'modules',
             'validator': str},
+        'pagure_flatpak_main_branch': {
+            'value': 'master',
+            'validator': str},
+        'pagure_module_main_branch': {
+            'value': 'master',
```

Comment:
By the time this code lands in production, the default branch should be "main", should we adjust the default value already?

Reply:
I've read that this has been reverted back to `master`, if I've not misunderstood

Reply:
For modules yes, for flatpak it's "stable', for container and rpm it's "rawhide"